rbkit-lib/model/cpustorage.cpp

```diff
@@ -3,6 +3,11 @@
 #include <QDebug>
 #include "cpumapping.h"
 
+RBKit::CpuStorage::CpuStorage()
```

Comment:
but the constructor is declared private in [cpustorage.h](https://github.com/code-mancers/rbkit-client/blob/cpu-treeview/rbkit-lib/model/cpustorage.h#L49), and we want to keep it that way so that one cannot instantiate another cpustorage, right?